book/src/template_syntax.md

```diff
@@ -584,6 +584,11 @@ let t = RenderInPlace { s1: SectionOne { a: "a", b: "b" } };
 assert_eq!(t.render().unwrap(), "Section 1: A=a\nB=b")
 ```
 
+**Note that if your inner template** like `SectionOne` **renders HTML content, you may want to
+disable escaping** when injecting it into an outer template, e.g. `{{ s1|escape("none") }}`;
```

Comment:
I'd use `safe` instead of `escape("none")`, which is a little more concise.

Reply:
Done